File: SAV_Backend/SAV_Backend/Models/Article.cs
﻿using System.Text.Json.Serialization;

namespace SAV_Backend.Models
{
    public class Article
    {
        public int Id { get; set; }
        public string Nom { get; set; }
        public string Description { get; set; }
        public DateTime DateFabrication { get; set; }
        public virtual ICollection<ClientArticle>? ClientArticles { get; set; }
        public ICollection<Reclamation>? Reclamations { get; set; }


    }

}

File: SAV_Frontend/src/Models/Article.ts
import { ClientArticle } from './ClientArticle';
import { Reclamation } from './Reclamation';

export interface Article {
  id: number;
  nom: string;
  description: string;
  dateFabrication: string;
  clientArticles?: ClientArticle[];
  reclamations: Reclamation[];
}
